src/misc/lv_style.h

```diff
@@ -154,6 +155,7 @@ typedef struct {
 
 /** A descriptor of a gradient. */
 typedef struct {
+    lv_area_t            grad_area;                    /**< gradient area */
```

Comment:
Is it something the user can set externally in a style? If so can we say `0;0;0;0` means "set the size automatically to the widget area" and handle it [here](https://github.com/lvgl/lvgl/blob/master/src/core/lv_obj_draw.c#L65)?

Reply:
Good idea. I would like to commit a patch to achieve this feature.